lib/Kind.py

```diff
@@ -1,38 +1,43 @@
 import common
 import time
 import os
+from robot.api import logger
+import configs
 
-DOCKER_HUB_REPO='kindest/node'
+DOCKER_HUB_REPO = 'kindest/node'
 CLUSTER_PREFIX = 'helm-acceptance-test'
 LOG_LEVEL = 'debug'
 
-MAX_WAIT_KIND_NODE_SECONDS = 60
+MAX_WAIT_KIND_NODE_SECONDS = 120
```

Comment:
why changing these values here?

Reply:
Sometimes it takes more than 60 secs to start up the cluster, most of the times cases fails because of `timeout`.

Reply:
may be we can add a variable option to the robot command rather than adding this hard coded?

Reply:
+1

Reply:
all valid points, but let's focus here on the main task - enabling running tests on an arbitrary cluster. Let's be as specific as possible - this would increase our chances to merge it upstream.

Reply:
Noted